drivers/media/i2c/imx477.c

```diff
@@ -1833,6 +1836,8 @@ static int imx477_start_streaming(struct imx477 *imx477)
 
 	/* Set vsync trigger mode: 0=standalone, 1=source, 2=sink */
 	tm = (imx477->trigger_mode_of >= 0) ? imx477->trigger_mode_of : trigger_mode;
+	imx477_write_reg(imx477, IMX477_REG_TEMP_SEN_CTL,
```

Comment:
Seeing as the enable is conditional at line 1855, make this line conditional too?
We've already enabled it in `mode_common_regs` otherwise.

Reply:
I was trying to cover the case where we don't re-send the common registers (e.g. on a mode switch).

OTOH keeping that write in the common setup is an attempt to ensure we never get a frame with a spurious temperature of zero.

Reply:
Ah, but I see it is safe to make them both conditional in that case